keps/sig-network/4427-relaxed-dns-search-validation/kep.yaml

```diff
@@ -4,19 +4,19 @@ authors:
   - "@sethev"
   - "@adrianmoisey"
 owning-sig: sig-network
-status: implementable
+status: implemented
```

Comment:
i forget when you're supposed to change this, but I guess the PRR reviewer will complain if it's wrong :slightly_smiling_face:

Reply:
I've always set this to implemented after the feature goes GA, but I'm happy to defer to @kubernetes/sig-release.

(I don't consider this field to be under PRR purview. we focus on the PRR section of the readme)